pages/in/widget/user/delete.vue

```diff
@@ -89,6 +97,9 @@ export default {
       'getAuthCoreNeedReAuth',
       'getAuthCoreAccessToken',
     ]),
+    targetConfirmMessage() {
+      return 'DELETE';
```

Comment:
Seems static, should be in data instead of computed?

Reply:
`data` is reactive, and can be changed
`computed` is more static when writing in such way



> This also means the following computed property will never update, because Date.now() is not a reactive dependency:

```js
computed: {
  now: function () {
    return Date.now()
  }
}
```

Reference: https://v2.vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods

Reply:
this specific example is a method though
i guess it's ok